writeBoundingBox.go

```diff
@@ -35,15 +35,35 @@ import (
 	"github.com/jolibrain/godd"
 )
 
-func writeBoundingBox(img image.Image, result godd.PredictResult, class int, ID string) (imgRGBA *image.RGBA) {
+func writeBoundingBox(img image.Image, result godd.PredictResult, class int, ID string, index int) (imgRGBA *image.RGBA) {
 
 	// Convert to RGBA
 	b := img.Bounds()
 	imgRGBA = image.NewRGBA(image.Rect(0, 0, b.Dx(), b.Dy()))
 	draw.Draw(imgRGBA, imgRGBA.Bounds(), img, b.Min, draw.Src)
 
-	// Set colors for bbox
-	red := color.RGBA{255, 0, 0, 255}
+  // Set colors for bbox
```

Comment:
Doesn't scale to the 20 classes of the voc model we use as default model. Insteady, I'd recommend putting your color settings into an array (of size 7, that's fine), and rotate around the array as needed. This means, for class say 17, 17 % 7 = 2, and remainder is 3, which would give the color for class 17. It doesn't matter much if some bbox of different classes have same color sometimes.